sysconfig/cloudinit.go

```diff
@@ -112,12 +111,19 @@ func configureCloudInit(opts *Options) (err error) {
 
 	// next check if there is a gadget cloud.conf to install
 	if HasGadgetCloudConf(opts.GadgetDir) {
-		// then copy / install the gadget config and return without considering
-		// CloudInitSrcDir
-		// TODO:UC20: we may eventually want to consider both CloudInitSrcDir
-		// and the gadget cloud.conf so returning here may be wrong
+		// then copy / install the gadget config first
 		gadgetCloudConf := filepath.Join(opts.GadgetDir, "cloud.conf")
-		return installGadgetCloudInitCfg(gadgetCloudConf, WritableDefaultsDir(opts.TargetRootDir))
+		err := installGadgetCloudInitCfg(gadgetCloudConf, WritableDefaultsDir(opts.TargetRootDir))
+		if err != nil {
```

Comment:
(nitpick^2) this could be a single line: `if err := install...; err != nil {`

Reply:
sure, the line is now 116 characters long but okay :smiley: